pkg/descheduler/controllers/migration/controller.go

```diff
@@ -108,7 +100,26 @@ func New(args runtime.Object, handle framework.Handle) (framework.Plugin, error)
 		return nil, err
 	}
 
-	if err = c.Watch(&source.Kind{Type: &sev1alpha1.PodMigrationJob{}}, &handler.EnqueueRequestForObject{}, &predicate.Funcs{
+	// New filter
```

Comment:
This meaningless comment should not exist.

Reply:
yes, I forget to delete them.